org.blackcoinmore.blackmore-qt.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id": "org.blackcoinmore.blackmore-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "command": "blackmore-qt",
+    "finish-args": [
+        "--socket=x11",
```

Comment:
it doesn't support Wayland?

Reply:
yes, it should support wayland

Reply:
sorry, I double checked. It doesn´t support wayland yet